tests/FlatBuffers.Test.Swift/Tests/FlatBuffers.Test.SwiftTests/monster_test_generated.swift

```diff
@@ -859,7 +859,9 @@ public struct MyGame_Example_Monster: FlatBufferObject, ObjectAPIPacker {
   public var anyAmbiguousType: MyGame_Example_AnyAmbiguousAliases { let o = _accessor.offset(VTOFFSET.anyAmbiguousType.v); return o == 0 ? .none_ : MyGame_Example_AnyAmbiguousAliases(rawValue: _accessor.readBuffer(of: UInt8.self, at: o)) ?? .none_ }
   public func anyAmbiguous<T: FlatBufferObject>(type: T.Type) -> T? { let o = _accessor.offset(VTOFFSET.anyAmbiguous.v); return o == 0 ? nil : _accessor.union(o) }
   public var vectorOfEnumsCount: Int32 { let o = _accessor.offset(VTOFFSET.vectorOfEnums.v); return o == 0 ? 0 : _accessor.vector(count: o) }
-  public func vectorOfEnums(at index: Int32) -> MyGame_Example_Color? { let o = _accessor.offset(VTOFFSET.vectorOfEnums.v); return o == 0 ? MyGame_Example_Color.red : MyGame_Example_Color(rawValue: _accessor.directRead(of: UInt8.self, offset: _accessor.vector(at: o) + index * 1)) }
```

Comment:
Enums in swift isn't completely a problem, since when we actually fetch the values we fetch them as Integers and then use the `rawValue` initializer to map the integers to actual swift enums. That's why we disabled the return of entire vectors. 
I see two ways of moving forward here, 
1- Implementing it C style. By using a swift struct, that should definitely be possible.
```swift
struct Color: Enum {
 public static var red = Color(0)
 ...
 var value: Uint8
 
 var named: String {
   switch { ... }
 }
}

```

2- We create `return _accessor.getVectorOfEnums(at: VTOFFSET.vectorOfEnums.v)` which  returns the numeric values of enums and then we can use map to convert the values to enums. This would affect the performance for sure.

Depending on what we want moving forward, the swift implementation would change to adapt to it

Reply:
Ok, so for the purposes of this PR, I'll just remove "returning the entire vector" from the API. Its up to you if you want to add that in the future or change how enums are modelled.

Reply:
Sounds good